ml_tools/model/prediction_strategy.py

```diff
@@ -159,6 +159,19 @@ def base_load_model(self, h5_file: h5py.File) -> None:
         self.input_features = input_features
 
 
+    @classmethod
+    @abstractmethod
+    def read_from_file(cls, file_name: str) -> None:
```

Comment:
Check to make sure we shouldn't be keeping the signatures consistent across abstract and child classes (i.e. should they not all return a `PredictionStrategy`?)

Reply:
Fixed